tasks/pipeline-configure.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+# dashboard-configure
+
+# Create api-key if not defined
+- set_fact: archivematica_src_configure_am_api_key={{ 999999999999999999998 | random | to_uuid | hash('md5') }}
+  when: archivematica_src_configure_am_api_key is undefined
+
+- name: "Create Dashboard user and register pipeline on SS"
```

Comment:
You have `archivematica_src_configure_dashboard` and `archivematica_src_configure_ss_dashboard: "no"`, but seems  like this configures both things always.